routing_common/transit_serdes.hpp

```diff
@@ -79,8 +81,12 @@ class Serializer
     t.Visit(*this);
   }
 
+  /// \note This method should be called beforw serializing every table.
+  void ResetCache() { m_lastEncodedPoint = m2::PointD(); }
```

Comment:
This is not a cache. Please, rename to Reset().

Reply:
Done